open-metadata-resources/open-metadata-deployment/docker/egeria-uistatic/pom.xml

```diff
@@ -0,0 +1,146 @@
+                        <phase>install</phase>
+                        <goals>
+                            <goal>tag</goal>
+                        </goals>
+                        <configuration>
+                            <tag>${project.version}</tag>
+                        </configuration>
+                    </execution>
+
+                    <execution>
+                        <id>push-version</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <tag>${project.version}</tag>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <repository>${docker.registry}/${docker.repo}/egeria-uistatic</repository>
+                    <buildArgs>
+                        <VCS_REF>${git.commit.id.describe}</VCS_REF>
+                        <VCS_DATE>${git.commit.time}</VCS_DATE>
+                        <BUILD_TIME>${git.build.time}</BUILD_TIME>
+                        <VCS_ORIGIN>${git.remote.origin.url}</VCS_ORIGIN>
+                    </buildArgs>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>tag-latest</id>
+            <activation>
+                <property>
+                    <name>docker.taglatest</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.spotify</groupId>
+                        <artifactId>dockerfile-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>tag-latest</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>tag</goal>
+                                </goals>
+                                <configuration>
+                                    <tag>latest</tag>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>push-latest</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>push</goal>
+                                </goals>
+                                <configuration>
+                                    <tag>latest</tag>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+</project>
```

Comment:
@planetf1 why is maven needed for this?

Reply:
Only because that is how we currently drive all our docker builds, and we already have the build pipelines setup. In some of the images we also have to retrieve files from other maven artifacts in the build, which the maven resources plugin manages.

There is no intrinsic need to use maven to build docker, and indeed it adds extra complexity.

I wanted to get something in and working quickly ... so to to stabilize master. Then work on improvements. One of those is that I think the build of the image should be done from sources within the egeria-ui project -- and we can have a simple build pipeline potentially that just uses the Dockerfile directly. That way we can easily trigger off changes and keep the image updated.

Reply:
i see, i have no problem with that, agree. i was imagining that the Dockerfile would be in the `odpi/egeria-ui` repository, but for the quickest way of stabilizing the master looks awesome so far, great work!